timetable/Application/Main.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System.Linq;
+using System.Xml.Linq;
+using Timetable.timetable.DB;
+using Timetable.timetable.Objects;
+using Timetable.timetable.XML;
+
+namespace Timetable
+{
+    public class Application
+    {
+		public static void Main()
```

Comment:
Shouldn't put this in Main, but in a separate FET file generator class. Is ok for now, will fix in #63.

Reply:
agree